clients/lsp-astro.el

```diff
@@ -25,6 +25,14 @@
 ;;; Code:
 (require 'lsp-mode)
 
+
+(defun lsp-astro--get-initialization-options ()
+  "Try to get the typescript server path, to supply to the astro language server."
+  (let ((library (f-join (lsp-workspace-root) "node_modules/typescript/lib/tsserverlibrary.js")))
+    (if (file-exists-p library)
+        `(:typescript (:serverPath ,library))
+      (lsp-warn "Unable to find typescript server path for astro-ls. Guessed: %s" library))))
```

Comment:
I assume that it may work without having tsserverlibrary specified, right?

Reply:
yup, the server then logs to stdio that it wont enable any typescript services. the behavior of this pr is taken from the astro vscode plugin (regarding the initialization options)

Reply:
Ok, does it make sense to have a variable to point it to custom location for the tsserver?

Reply:
I guess not really because it is supposed to take the typescript in the version bundled by the project (if you use typescript you will have it as a dev dependency). And then it should always pickup that specific tsserver library.

Again, afaik, this is done equally in the vscode extension that is developed alongside the language server.

Reply:
Ok, although it looks strange to me because it seems like there is no point to have that logic in the client - if you are going to look in hardcoded location then equally well you can have it in the server.

Reply:
yeah seems to be an issue on their side. Maybe one could contribute upstream.

But thanks for the review 🎉